### ISSUE TITLE
Chrome/Chromium does not use system FreeType

### ISSUE THREAD

**Issue by pdeljanov** [pdeljanov/infinality-remix#3]
Chrome and Chromium-based browsers or applications do not use the system FreeType library. Therefore these applications don't get the Infinality improvements. There isn't much that can be done unfortunately other than compiling Chromium using the system FreeType library. This can be accomplished by setting the build flag: `use_system_freetype=true`.

I will consider adding a Chromium PKGBUILD in here to support this, but compiling Chromium is generally not feasible unless you have an extremely strong computer.

**Comment by TomaszGasior** [comment]
`chromium` package on Arch official repo is affected too?

**Comment by pdeljanov** [comment]
> `chromium` package on Arch official repo is affected too?

Yes, no official or AUR chromium package is built with the `use_system_freetype=true` build flag. You could always modify the PKGBUILD for chromium, but the build will probably take the better part of a day...

An alternative is Firefox, which does use the system's FreeType library (for now), but it has other font rendering problems unrelated to infinality-remix.

**Comment by jstkdng** [comment]
>no official or AUR chromium package is built with the use_system_freetype=true build flag

hey now, ```ungoogled-chromium-bin``` *is* built with the ```use_system_freetype=true``` flag.

>but compiling Chromium is generally not feasible unless you have an extremely strong computer.

can confirm this, but you can also use OpenSUSE's Build Service, I use it to compile ```ungoogled-chromium-bin``` https://build.opensuse.org/package/show/home:justkidding:arch/ungoogled-chromium. 
really helps out a lot.

**Comment by Danhyal-Masood** [comment]
> > no official or AUR chromium package is built with the use_system_freetype=true build flag
> 
> hey now, `ungoogled-chromium-bin` _is_ built with the `use_system_freetype=true` flag.
> 
> > but compiling Chromium is generally not feasible unless you have an extremely strong computer.
> 
> can confirm this, but you can also use OpenSUSE's Build Service, I use it to compile `ungoogled-chromium-bin` https://build.opensuse.org/package/show/home:justkidding:arch/ungoogled-chromium.
> really helps out a lot.

Did you notice a difference? I compiled chromium with the flag and couldn't see any changes.

**Comment by pdeljanov** [comment]
> > no official or AUR chromium package is built with the use_system_freetype=true build flag
> 
> hey now, `ungoogled-chromium-bin` _is_ built with the `use_system_freetype=true` flag.

Thanks, good to know!

> Did you notice a difference? I compiled chromium with the flag and couldn't see any changes.

I personally run the official Chrome build, so I wouldn't be able to tell you. However, you could try comparing the same page on Firefox (71 or later) vs. chromium. I can confirm that Firefox renders as expected with Infinality.
